bin/buck

```diff
@@ -18,5 +18,5 @@ function abs_script_dir_path {
 
 BUCK_DIR=$(abs_script_dir_path "$0")
 
-PYTHON=$(command -v python2 python | head -1)
+PYTHON=$(command -v python2.7 python | head -1)
```

Comment:
This should be fine, but can you make this python2.7 python2 python in case, for some reason, a person doesn't have a 2.7 link?

Reply:
Done!